repository/src/main/java/org/apache/atlas/repository/store/graph/v2/preprocessor/AuthPolicyValidator.java

```diff
@@ -156,26 +138,36 @@ public void validate(AtlasEntity policy, AtlasEntity existingPolicy,
                     validateParam (!PERSONA_POLICY_VALID_SUB_CATEGORIES.contains(policySubCategory),
                             "Please provide valid value for attribute " + ATTR_POLICY_SUB_CATEGORY + ":"+ PERSONA_POLICY_VALID_SUB_CATEGORIES);
 
+
+                    List<String> resources = getPolicyResources(policy);
```

Comment:
Why empty resources should not be allowed?

Reply:
While creating policies we need policy resources